File: rust/art.rs
use std::io::File;

// "2 to the 5th" programming competition @ engineering.cerner.com
// 
// Read in the ASCII art file and simply print it to the console. 
// Note: This assumes you are executing "rustc art.rs && ./art" in 
// the current working directory.
//
// Reference: http://doc.rust-lang.org/std/io/
//
fn main() {
   let filepath = Path::new("../art/alan.txt");
   let contents = File::open(&filepath).read_to_string();
   println!("{}", contents);
}

File: c/art.c
#include <stdio.h>

// "2 to the 5th" programming competition @ engineering.cerner.com
// 
// Read in the ASCII art file and simply print it to the console. 
// Note: This assumes you are executing "gcc art.c -o art && ./art" 
// in the current working directory.
//
// Reference: http://stackoverflow.com/questions/14062910/clearest-way-to-read-and-print-txt-file-lines-in-c
//
int main(int argc, char **argv)
{
   FILE* filePtr;
   const char* fileName = "../art/alan.txt";
   if ((filePtr = fopen(fileName, "r")) != 0) {
      char* buffer = 0;
      size_t bufferLength = 0;
      while (getline(&buffer, &bufferLength, filePtr) != -1)
         fputs(buffer, stdout);

      // Clean up
      fclose(filePtr);
   }
   return(0);
}